certif/app/controllers/authenticated/team/invite.js

```diff
@@ -2,19 +2,90 @@ import Controller from '@ember/controller';
 import { service } from '@ember/service';
 import { action } from '@ember/object';
 
+import ENV from 'pix-certif/config/environment';
+import get from 'lodash/get';
+
 export default class AuthenticatedTeamInviteController extends Controller {
+  @service currentUser;
+  @service intl;
+  @service notifications;
   @service router;
+  @service store;
+
+  email = null;
 
   @action
   async createCertificationCenterInvitation(event) {
     event.preventDefault();
+
+    const certificationCenterId = this.currentUser.currentAllowedCertificationCenterAccess.id;
+    const emailsTextareaInput = Object.values(event.target)
+      .filter((target) => target.type === 'textarea')
+      .shift();
+    const emails = emailsTextareaInput.value.replace(/[\s\r\n]/g, '').split(',');
+
+    try {
+      await this.store.adapterFor('certification-center-invitation').sendInvitations({ certificationCenterId, emails });
+
+      const message =
+        emails.length > 1
+          ? this.intl.t('pages.team-invite.notifications.success.invitations')
+          : this.intl.t('pages.team-invite.notifications.success.invitation', { email: emails.shift() });
+
+      this.notifications.success(message);
+      this.router.transitionTo('authenticated.team.list.invitations');
+    } catch (responseError) {
+      if (responseError.errors) {
+        const errorMessage = this._handleApiError(responseError);
+        this.notifications.error(errorMessage);
+      } else {
+        // eslint-disable-next-line no-console
+        console.error(responseError);
+        this.notifications.error(this.intl.t('common.form-errors.default'));
+      }
+    }
   }
 
   @action
-  updateEmail() {}
+  updateEmail(event) {
+    this.email = event?.target?.value?.trim();
```

Comment:
```suggestion
    this.email = event.target?.value?.trim();
```

En effet le premier `?.` est inutile dans ce cas cf. https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Optional_chaining